### DISCUSSION TITLE
Azure DevOps Server (on-prem) support

### DISCUSSION THREAD

**Discussion by clizzotteUNUM** [microsoft/vsts-extension-retrospectives#128] [General]
I see where this supports Azure DevOps but it does not differentiate between the SaaS offering and the on premises Server install.
I am able to install the extension onto our Azure DevOps Server 2020 installation but when we try to use the retrospective boards, it just clocks and eventually times out. Is there a document somewhere that might go through any network or firewall configurations that are needed to operate in a corporate network install? Are there sites to whitelist?

**Comment by ghost** [comment]
I am also using with Server 2020 and don't have these issues. It worked immediately after installation.

**Reply by mcvermeulen** [reply]
It does work on our on-prem. 
However the extension uses a CDN for fonts. And since we don't have any internet connection, we had to change and build the extension to use our own fontfiles

**Comment by ozdemiremrah81** [comment]
Hi, i just want to share that, pictures are not showing on following webpage. 
https://marketplace.visualstudio.com/items?itemName=ms-devlabs.team-retrospectives
